collection/collection/src/commonMain/kotlin/androidx/collection/SparseArrayCompat.kt

```diff
@@ -240,24 +240,25 @@ public expect open class SparseArrayCompat<E>
 }
 
 @Suppress("NOTHING_TO_INLINE")
-private inline fun <E, T : E?> SparseArrayCompat<E>.internalGet(key: Int, defaultValue: T): T {
+// TODO: revert the change: this function was changed in JB fork because of https://youtrack.jetbrains.com/issue/KT-65061
+private inline fun SparseArrayCompat<*>.internalGet(key: Int, defaultValue: Any?): Any? {
```

Comment:
It will cause a conflict with integration branch. I fixed it there by https://github.com/JetBrains/compose-multiplatform-core/commit/38b8a078a73854eb0a7ba64ead4e4310dad0c5bd
Will that fix work on jb-main too?

Reply:
yes, your fix helps in jb-main too. 

applied it here.